tests/integration/targets/aws_eks_cluster/tasks/full_test.yml

```diff
@@ -160,6 +160,28 @@
         that:
           - eks_delete is changed
 
+    - name: create EKS cluster with short name
+      aws_eks_cluster:
+        name: "{{ eks_cluster_short_name }}"
+        security_groups: "{{ eks_security_groups | community.general.json_query('[].name') }}"
```

Comment:
Could you please remove the dependency on community.general and switch to selectattr?